docs/source/access-data/tutorials/initial-setup.rst

```diff
@@ -3,194 +3,189 @@ Initial Setup
 
 **Learning Objectives:**
 
-- Complete the initial setup for data access
+Complete the initial setup required for data access:
```

Comment:
I don't think this list is necessary since it's essentially a duplicate of the sidebar menu.

Reply:
Michael recommended putting learning objectives at the top of each tutorial. Many will link directly to sections, but not all sections will be represented in the learning objectives. I think some duplication here is ok.